src/fstests/fstests.jinja2

```diff
@@ -24,31 +24,37 @@ class Job(BaseJob):
         if os.path.exists(self._workspace):
             shutil.rmtree(self._workspace)
 
+    def _generate_node_result(self, name, status):
```

Comment:
This function can be used to add state and result.
```suggestion
    def _generate_node_result(self, name, state=None, result=None):
```
The keys `state` and `result` will be included in the dictionary only if the value is not `None`.
Then the function can be used in `_run` method too.

Reply:
updated.

Reply:
The `state` and `result` should be added to the `node` dict only if provided. Otherwise, the values will be reset to null.

Reply:
Where is the `_generate_node_result` being used with the `state` param?

Reply:
hey @JenySadadia, here:
https://github.com/kernelci/kernelci-pipeline/pull/156/files#diff-b8768ab4609a0556f87a0d84b3de88b747b1d0698d018af1128c9362b97a9073R145